resources/fixtures/American_DJ/American-DJ-Inno-Pocket-Spot-Twins.qxf

```diff
@@ -14,24 +14,24 @@
  <Channel Name="Gobo Wheel 1">
   <Group Byte="0">Gobo</Group>
   <Capability Min="0" Max="7" Preset="GoboMacro" Res1="Others/open.svg">Open</Capability>
-  <Capability Min="8" Max="15">Gobo 1</Capability>
-  <Capability Min="16" Max="23">Gobo 2</Capability>
-  <Capability Min="24" Max="31">Gobo 3</Capability>
-  <Capability Min="32" Max="39">Gobo 4</Capability>
-  <Capability Min="40" Max="47">Gobo 5</Capability>
-  <Capability Min="48" Max="55">Gobo 6</Capability>
-  <Capability Min="56" Max="63">Gobo 7</Capability>
+  <Capability Min="8" Max="15" Preset="GoboMacro" Res1="">Gobo 1</Capability>
```

Comment:
Without a picture, a preset alone doesn't make much sense for preview

Reply:
This is the agreed approach: https://www.qlcplus.org/forum/viewtopic.php?f=35&t=14353&start=10#p60205

Reply:
Ok, I still don't like it but if we agreed it then let's proceed
I need to check what happen in QLC+ 5 with no picture at all